WalletWasabi.Fluent/Models/Wallets/TransactionModel.cs

```diff
@@ -9,33 +9,33 @@ public partial class TransactionModel : ReactiveObject
 {
 	private readonly List<TransactionModel> _children = new();
 
-	public required int OrderIndex { get; init; }
+	public int OrderIndex { get; init; }
 
-	public required uint256 Id { get; init; }
+	public uint256 Id { get; init; }
 
-	public required LabelsArray Labels { get; init; }
+	public LabelsArray Labels { get; init; }
 
-	public required DateTimeOffset Date { get; set; }
+	public DateTimeOffset Date { get; set; }
 
-	public required string DateString { get; set; }
+	public string DateString { get; set; }
 
-	public required int Confirmations { get; init; }
+	public int Confirmations { get; init; }
 
 	public int BlockHeight { get; init; }
 
 	public uint256? BlockHash { get; init; }
 
-	public required string ConfirmedTooltip { get; set; }
+	public string ConfirmedTooltip { get; set; }
 
-	public required TransactionType Type { get; init; }
+	public TransactionType Type { get; init; }
 
-	public required TransactionStatus Status { get; set; }
+	public TransactionStatus Status { get; set; }
 
 	public bool IsChild { get; set; }
 
 	public Money? Balance { get; set; }
 
-	public required Money Amount { get; set; }
+	public Money Amount { get; set; }
```

Comment:
WTF!?!?! 
When I press CTRL+S to save the document, it automatically deletes the `required` words...